<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>music</class>
 <widget class="QWidget" name="music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>804</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>804</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/音乐播放器图标.png</normaloff>:/image/音乐播放器图标.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>781</width>
     <height>94</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {  
    height: 10px; /* 滑槽的高度 */  
    background: rgba(255, 255, 255, 0.3); /* 浅白色半透明背景 */  
    border: 1px solid rgba(0, 0, 0, 0.1); /* 轻微黑色边框 */  
    border-radius: 5px; /* 圆角边框 */  
    margin: 0 10px; /* 边缘间距 */  
}  
  
QSlider::handle:horizontal {  
    background: rgba(100, 149, 237, 0.8); /* 滑块颜色，浅蓝色半透明 */  
    width: 20px; /* 滑块宽度 */  
    height: 20px; /* 滑块高度 */  
    margin: -5px 0; /* 滑块相对于滑槽的垂直位置调整 */  
    border-radius: 10px; /* 滑块圆角 */  
    border: 2px solid rgba(255, 255, 255, 0.5); /* 滑块边框，白色半透明 */  
}  
  
QSlider::add-page:horizontal {  
    background: rgba(150, 200, 255, 0.2); /* 已选区域颜色，浅蓝色半透明 */  
    border-radius: 5px; /* 圆角 */  
}  
  
QSlider::sub-page:horizontal:disabled {  
    background: none; /* 禁用时未选区域不显示背景 */  
}  
  
QSlider::add-page:horizontal:disabled {  
    background: rgba(100, 100, 100, 0.2); /* 禁用时已选区域颜色，灰色半透明 */  
} </string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {  
    background-color: rgba(255, 255, 255, 128); /* 半透明白色背景 */  
    border: 2px solid black; /* 黑色边框 */  
    border-radius: 25px; /* 圆形边框 */  
    padding: 5px; /* 内边距 */  
}  
QPushButton:hover {  
    background-color: rgba(255, 0, 0, 128); /* 鼠标悬停时变为半透明红色 */  
}  
QPushButton:pressed {  
    background-color: rgba(0, 0, 255, 128); /* 鼠标按下时变为半透明蓝色 */  
    border-style: inset; /* 按下时边框样式变为内嵌 */  
}</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);
border-radius: 30px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/image/上一首.png</normaloff>:/image/上一首.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
border-radius: 30px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);
border-radius: 30px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/image/下一首.png</normaloff>:/image/下一首.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {  
    height: 10px; /* 滑槽的高度 */  
    background: rgba(255, 255, 255, 0.3); /* 浅白色半透明背景 */  
    border: 1px solid rgba(0, 0, 0, 0.1); /* 轻微黑色边框 */  
    border-radius: 5px; /* 圆角边框 */  
    margin: 0 10px; /* 边缘间距 */  
}  
  
QSlider::handle:horizontal {  
    background: rgba(100, 149, 237, 0.8); /* 滑块颜色，浅蓝色半透明 */  
    width: 20px; /* 滑块宽度 */  
    height: 20px; /* 滑块高度 */  
    margin: -5px 0; /* 滑块相对于滑槽的垂直位置调整 */  
    border-radius: 10px; /* 滑块圆角 */  
    border: 2px solid rgba(255, 255, 255, 0.5); /* 滑块边框，白色半透明 */  
}  
  
QSlider::add-page:horizontal {  
    background: rgba(150, 200, 255, 0.2); /* 已选区域颜色，浅蓝色半透明 */  
    border-radius: 5px; /* 圆角 */  
}  
  
QSlider::sub-page:horizontal:disabled {  
    background: none; /* 禁用时未选区域不显示背景 */  
}  
  
QSlider::add-page:horizontal:disabled {  
    background: rgba(100, 100, 100, 0.2); /* 禁用时已选区域颜色，灰色半透明 */  
} </string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>782</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
    QTableView {  
        background-color: rgba(255, 255, 255, 100); /* 半透明背景 */  
        selection-background-color: yellow; /* 选中时的背景颜色 */  
        selection-color: black; /* 选中时的文字颜色（如果需要的话） */  
        show-grid: false; /* 但这通常不直接控制列线，因为列线是由边框或间隔控制的 */  
        border: 0px; /* 去除表格的边框 */  
        gridline-color: transparent; /* 尝试去除网格线，但这可能不起作用，因为网格线通常不是由这个属性控制的 */  
    }  
  
    QTableView::item {  
        border: 0px; /* 去除单元格的边框 */  
    }</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,128);
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView_2">
      <property name="styleSheet">
       <string notr="true">
    QTableView {  
        background-color: rgba(255, 255, 255, 100); /* 半透明背景 */  
        selection-background-color: yellow; /* 选中时的背景颜色 */  
        selection-color: black; /* 选中时的文字颜色（如果需要的话） */  
        show-grid: false; /* 但这通常不直接控制列线，因为列线是由边框或间隔控制的 */  
        border: 0px; /* 去除表格的边框 */  
        gridline-color: transparent; /* 尝试去除网格线，但这可能不起作用，因为网格线通常不是由这个属性控制的 */  
    }  
  
    QTableView::item {  
        border: 0px; /* 去除单元格的边框 */  
    }</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>781</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);
border-radius: 30px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/image/音乐播放器图标.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,50);</string>
       </property>
       <property name="placeholderText">
        <string>搜索歌手/歌曲</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,50);</string>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>781</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
background-color: rgba(255, 170, 255, 70);</string>
    </property>
    <property name="text">
     <string> 当前播放歌曲歌词</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
background-color: rgba(255, 170, 255, 70);</string>
    </property>
    <property name="text">
     <string> 历史歌曲列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
background-color: rgba(255, 170, 255, 70);</string>
    </property>
    <property name="text">
     <string> 搜索歌曲列表</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
